code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -276,9 +276,9 @@
 				if(prob(70))
 					user << "<span class='notice'>You remove the turret and salvage some components.</span>"
 					if(installation)
-						var/obj/item/weapon/gun/energy/Gun = new installation(loc)

```

Comment:
such a great var name, when you know there was an actual gun pathed as "/obj/item/weapon/gun/energy/gun". Jesus...